ds701_book/20-Recommender-Systems.qmd

```diff
@@ -23,27 +22,28 @@ import laUtilities as ut
 %matplotlib inline
 ```
 
-
-
 Today, we look at a topic that has become enormously important in society: recommender systems.
 
 We will
+
 * Define recommender systems
 * Review the challenges they pose
 * Discuss two classic methods:
     * Collaborative Filtering
     * Matrix Factorization
 
-```{margin}
-This section draws heavily on 
+::: aside
```

Comment:
This aside shows up poorly in the slides.

![Screenshot 2024-11-15 at 11 02 07 AM](https://github.com/user-attachments/assets/60223b9e-c5d2-4b15-9c5f-e75288126ac3)